### PR TITLE
documentation on types of STARDIS returns (model, plasma, rad-field)

### PR DESCRIPTION
**Pull request by RyanGroneck**
### :pencil: Description

**Type:**  :memo: `documentation`

This is the draft documentation for explaining the return options of STARDIS, describing and giving examples of Stellar_Model, Stellar_Plasma, and Stellar_Radiation_Field use cases.

Also, link issues affected by this pull request by using the keywords: `close`, `closes`, `closed`, `fix`, `fixes`, `fixed`, `resolve`, `resolves` or `resolved`. 

### :pushpin: Resources

Examples, notebooks, and links to useful references.

### :vertical_traffic_light: Testing

How did you test these changes?

- [ ] Testing pipeline
- [ ] Other method (describe)
- [ ] My changes can't be tested (explain why)

### :ballot_box_with_check: Checklist

- [ ] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

> **Note:** If you are not allowed to perform any of these actions, ping (@) a contributor.

### CODE REVIEW

File: docs/possible_stardis_outputs/possible_stardis_outputs.ipynb [hunk 1]
```diff
@@ -0,0 +1,746 @@
+{
```
Comment (jvshields) [inline docs/possible_stardis_outputs/possible_stardis_outputs.ipynb:1]: <p>Careful about your imports. Not sure you use download_atom_data anywhere in this notebook. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/stardis/pull/230/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (jvshields) [inline docs/possible_stardis_outputs/possible_stardis_outputs.ipynb:1]: <p>Probably easier to just show the shape of one of these things, and say generically that they are of shape, (depth_point_index, wavelength_index)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/stardis/pull/230/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (jvshields) [inline docs/possible_stardis_outputs/possible_stardis_outputs.ipynb:1]: <p>We talked a bit about this, but it's probably better to just show a plot demonstrating what these things are than printing out the very large dictionary with very many numbers. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/stardis/pull/230/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (RyanGroneck) [reply docs/possible_stardis_outputs/possible_stardis_outputs.ipynb:1]: Addressed in '[stellar_radiation and import fix](https://github.com/tardis-sn/stardis/pull/230/commits/da2fff3e2f6b866af6ebeff0ef9e3499922d21e5)'

Comment (RyanGroneck) [reply docs/possible_stardis_outputs/possible_stardis_outputs.ipynb:1]: Addressed in '[stellar_radiation and import fix](https://github.com/tardis-sn/stardis/pull/230/commits/da2fff3e2f6b866af6ebeff0ef9e3499922d21e5)'

Comment (RyanGroneck) [reply docs/possible_stardis_outputs/possible_stardis_outputs.ipynb:1]: Addressed in '[stellar_radiation and import fix](https://github.com/tardis-sn/stardis/pull/230/commits/da2fff3e2f6b866af6ebeff0ef9e3499922d21e5)'

**Review by jvshields — APPROVED** [review]

**Review by andrewfullard — APPROVED** [review]

**Pull request merged**